### PR TITLE
feat(2580): Managing child pipelines

### PR DESCRIPTION
**Pull request by sagar1312**
## Context

When a SCM URL associated with a child pipeline is removed from the parent configuration, the child pipeline in not auto deleted. Instead the child pipeline is marked as inactive, allowing the parent pipeline admin to below actions
* reactivate the pipeline by restoring the SCM URL in the parent pipeline configuration
* explicitly delete the pipeline from the UI/API

## Objective

This PR includes the documentation on managing child pipelines

## References

https://github.com/screwdriver-cd/screwdriver/issues/2580

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: docs/user-guide/configuration/externalConfig.md [hunk 1]
```diff
@@ -49,11 +51,31 @@ jobs:
 
 Builds for child pipeline will have access to parent pipeline's repository at [`$SD_CONFIG_DIR`](../environment-variables#directories).
 
+## Manage Child Pipelines
+When a child pipeline SCM URL is added to the parent pipeline configuration, child pipeline gets created with `ACTIVE` state.
+
+Child pipeline state will be set to `INACTIVE` when the associated SCM URL has been removed from the parent pipeline configuration.
```
Comment (jithine) [inline docs/user-guide/configuration/externalConfig.md]: should we mention, parent pipeline configuration in the screwdriver.yaml ?

Comment (sagar1312) [reply docs/user-guide/configuration/externalConfig.md]: Done

File: docs/user-guide/configuration/externalConfig.md [hunk 2]
```diff
@@ -49,11 +51,31 @@ jobs:
 
 Builds for child pipeline will have access to parent pipeline's repository at [`$SD_CONFIG_DIR`](../environment-variables#directories).
 
+## Manage Child Pipelines
+When a child pipeline SCM URL is added to the parent pipeline configuration, child pipeline gets created with `ACTIVE` state.
+
+sChild pipeline state will be set to `INACTIVE` when the associated SCM URL has been removed from the screwdriver.yaml of the parent pipeline.
+Pipeline admin can review and take necessary action on such inactive child pipeline
+* reactivate the pipeline by restoring the SCM URL in the screwdriver.yaml of the parent pipeline 
+* explicitly delete the pipeline from the UI/API
```
Comment (tkyi) [inline docs/user-guide/configuration/externalConfig.md]:
```suggestion
When a child pipeline SCM URL is added to the parent pipeline configuration, a child pipeline gets created with `ACTIVE` state.

When a child pipeline SCM URL is removed from the parent pipeline configuration, the child pipeline;s state will be set to `INACTIVE`.
Parent pipeline admins can review and take necessary action on these inactive child pipelines by either:
* reactivating the pipeline by restoring the SCM URL in the screwdriver.yaml of the parent pipeline 
* explicitly deleting the pipeline from the UI/API
```

**Comment by tkyi** [comment]
Some suggestions

File: docs/user-guide/configuration/externalConfig.md (hunk 2 above)
Comment (sagar1312) [reply docs/user-guide/configuration/externalConfig.md]: Done

**Review by tkyi — APPROVED** [review]

**Pull request merged**